### ISSUE TITLE
HTTP status 500 Internal Server Error

### ISSUE THREAD

**Issue by cdpixa2017** [sentinelsat/sentinelsat#237]
I am attempting to download data when I run across the following error `HTTP status 500 Internal Server Error: UnsupportedOperationException : Error creating stream from file`. Is this indicative of a server issue or something that is occurring due to a code issue?

**Comment by j08lue** [comment]
This is a server error. Is this on the S3 pre-ops hub? I am seeing tons of those there and already wrote to SciHub support, waiting for answer.

Please be more specific about the URL you are trying to download.

**Comment by cdpixa2017** [comment]
Yes it is on the S3 pre-ops hub and as it's a server error I'll wait on a response to that.  I'm trying to download some near real time data.

**Comment by j08lue** [comment]
From `EOSupport@copernicus.esa.int`:

> This relates to the recent news published on 26/09/2018 that OLCI data products have been moved to the Open Access Hub and API Hub: https://scihub.copernicus.eu/news/News00392. 
> 
> This news states that during this process S3A OLCI data has been re-processed to [Processing Baseline 2.23](https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-3-olci/processing-baseline). Re-processed products cover L1 and Land L2 in both Full and Reduced Resolution since 25 April 2016 to 30 November 2017. These products are identified by the filename ending with `'*_LR2_R_NT_002'` ; meaning R (re-processed) products at LR2 (Land Reprocessing Centre `#2`) belonging to product Collection Baseline `'002'`; contrary to the old products which ended normally by `'*_LN1_O_NT_001'`.
> 
> To confirm this, I searched the Open Access Hub using the footprint of the unavailable product: `S3A_OL_1_EFR____20170702T093613_20170702T093913_20170703T135643_0179_019_250_1980_LN1_O_NT_002` and the following data product was returned: `S3A_OL_1_EFR____20170702T093613_20170702T093913_20180505T104542_0179_019_250_1980_LR2_R_NT_002`

**Comment by j08lue** [comment]
tl;dr Use the regular SciHub and adjust your query to look for the re-processed products.

**Comment by j08lue** [comment]
https://scihub.copernicus.eu/news/News00392

**Comment by j08lue** [comment]
As I understand the news from September, the Open Access Hub might not yet have all the data that the S3 pre-ops hub had, because moving those apparently takes a long time for some reason. That is also what I am experiencing (more data in pre-ops hub than in OA hub). But the pre-ops hub does not have the re-processed data. 

So for the time being, it might be that you need to query both to get a full time series. I'll ask EOSupport for confirmation on this.

**Comment by cdpixa2017** [comment]
I'm not sure that is applicable as I am downloading data from the SLSTR instrument. Unless this data move has affected it in some way? Querying the products I am downloading show they are present and can be sent to a dataframe but the api error is returned  when attempting to download.

**Comment by valgur** [comment]
I'm closing this due to inactivity. This looks like more of a SciHub issue than something sentinelsat-specific anyway.

**Comment by valgur** [comment]
See also: #483.
